docs/providers/libvirt/r/domain.html.markdown

```diff
@@ -31,6 +31,14 @@ The following arguments are supported:
 * `vcpu` - (Optional) The amount of virtual CPUs. If not specified, a single CPU will be created.
 * `disk` - (Optional) An array of one or more disks to attach to the domain. The `disk` object structure is documented below.
 * `network_interface` - (Optional) An array of one or more network interfaces to attach to the domain. The `network_interface` object structure is documented below.
+* `metadata` - (Optional) A string containing valid XML data. This is going to be
+  added to the final domain inside of the [metadata tag](https://libvirt.org/formatdomain.html#elementsMetadata).
```

Comment:
You are right, I think this PR is somehow screwed up, there are some changes inside of it that have already been marged by you. I'm going to rebase it.